src/components/filter.rs

```diff
@@ -0,0 +1,75 @@
+use sycamore::prelude::*;
+
+// table structure
+#[component]
+pub fn Filter<'a, G: Html>(
+    cx: Scope<'a>,
+) -> View<G> {
+    view! { cx,
+        div (class="flex flex-col shadow-md rounded-md ") {
+            // filter title
+            div (class="bg-primary p-4 font-bold rounded-t-md") { 
+                p(class="text-primary-content text-lg") { "Filter" }
+            }
+        
+            // Term
+            div(class="flex rounded-b-md") {
+                FilterSection(title="Term".to_string())
+                
+            }
+            div(class="flex rounded-b-md") {
+                FilterSection(title="Time".to_string())
+                
+            }
+        }       
+    }        
+}
+
+// filter section
+#[derive(Prop)]
+pub struct FilterScetionProps {
+    title: String,
+    // TODO: allow multiple filter selections 
+    // names: Vec<String>,
+}
+
+#[component]
+pub fn FilterSection<'a, G: Html>(
+    cx: Scope<'a>,
+    FilterScetionProps { title }: FilterScetionProps,
+) -> View<G> {
+    view! {cx,
+        div(class="relative p-4 bg-base-100 rounded-b-md w-full") {
+            h3 (class="text-lg font-semibold") { (title) }
+            div (class="flex justify-center") {
+                div (class="flex flex-col w-1/2") {
+                    CheckBox(name="testing".to_string())
+                    CheckBox(name="testing".to_string())  
+                }
```

Comment:
Please use daisyui input classes 
<img width="308" alt="image" src="https://github.com/GauravTalreja/ebb/assets/59036640/699293cc-95a8-4d7c-bf55-24bd841f08d2">

Reply:
I added the container component form-control to CheckBox component, so now the CheckBox is the same as the one in daisyUI